LoopbackApp/app/build.gradle

```diff
@@ -1,40 +1,38 @@
-apply plugin: 'com.android.model.application'
+/**
+ * The default build is with CMake Plugin.
+ * If Android.mk build system is required, src/main/cpp/Android.mk is
+ * a workable, just add hooks to this file to replace cmake at the "cmake"
+ * location.
+ */
 
-model {
-    android {
+apply plugin: 'com.android.application'
+
+android {
         compileSdkVersion 26
-        buildToolsVersion '26.0.0'
 
         defaultConfig {
             applicationId 'org.drrickorang.loopback'
-            minSdkVersion.apiLevel 11
-            targetSdkVersion.apiLevel 26
+            minSdkVersion 23
```

Comment:
we still need to be able to run on older platforms

Reply:
yeah, but ATrace is not exposed from NDK until very recently, could you check you source code to see if you have platform guard ?  or use dlopen/dlclose to get get the symbol.  I did not understand why it could build previously. could check the source code? I am not familar with the source